dockertest/containers.py

```diff
@@ -563,14 +563,14 @@ def clean_all(self, containers):
             preserve_cnames = get_as_list(preserve_cnames)
         else:
             preserve_cnames = []
```

Comment:
R0204:566,8: DockerContainers.clean_all: Redefinition of preserve_cnames type from list to set